README.rst

```diff
@@ -58,8 +58,8 @@ starfish supports python 3.8-11 and can easily be installed from PyPI:
 Python Version Notice
```

Comment:
I think we can leave out this  "Python Version Notice" unless we see something imminent with 3.9 support going away.

Reply:
I wanted to discourage new users from picking python 3.9 for starfish as the scientific python community already recommended dropping support for python 3.9 at the end of 2023: https://scientific-python.org/specs/spec-0000/#2023---quarter-4
It is probably just a matter of time, when a critical security issue with a dependency will force us to drop python 3.9.

But I don't feel strongly about it, if you think it is better to take the warning out.